oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -384,15 +382,16 @@ receiver-otel:
 receiver-zipkin:
   selector: ${SW_RECEIVER_ZIPKIN:-}
   default:
-    host: ${SW_RECEIVER_ZIPKIN_HOST:0.0.0.0}
-    port: ${SW_RECEIVER_ZIPKIN_PORT:9411}
-    contextPath: ${SW_RECEIVER_ZIPKIN_CONTEXT_PATH:/}
-    jettyMinThreads: ${SW_RECEIVER_ZIPKIN_JETTY_MIN_THREADS:1}
-    jettyMaxThreads: ${SW_RECEIVER_ZIPKIN_JETTY_MAX_THREADS:200}
-    jettyIdleTimeOut: ${SW_RECEIVER_ZIPKIN_JETTY_IDLE_TIMEOUT:30000}
-    jettyAcceptorPriorityDelta: ${SW_RECEIVER_ZIPKIN_JETTY_DELTA:0}
-    jettyAcceptQueueSize: ${SW_RECEIVER_ZIPKIN_QUEUE_SIZE:0}
+    # For HTTP server
+    restHost: ${SW_RECEIVER_ZIPKIN_REST_HOST:0.0.0.0}
+    restPort: ${SW_RECEIVER_ZIPKIN_REST_PORT:9411}
+    restContextPath: ${SW_RECEIVER_ZIPKIN_REST_CONTEXT_PATH:/}
+    restMaxThreads: ${SW_RECEIVER_ZIPKIN_REST_MAX_THREADS:200}
+    restIdleTimeOut: ${SW_RECEIVER_ZIPKIN_REST_IDLE_TIMEOUT:30000}
+    restAcceptorPriorityDelta: ${SW_RECEIVER_ZIPKIN_REST_DELTA:0}
```

Comment:
I think this configuration is removed already?

Reply:
I just merged this a minute ago. Please submit a pull request to fix.

Reply:
https://github.com/apache/skywalking/blob/12ba7aa5336af9b323bb2acbc1d864b6b3c416df/oap-server/server-receiver-plugin/zipkin-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/zipkin/ZipkinReceiverProvider.java#L72

not removed for now in zipkin

Reply:
> https://github.com/apache/skywalking/blob/12ba7aa5336af9b323bb2acbc1d864b6b3c416df/oap-server/server-receiver-plugin/zipkin-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/zipkin/ZipkinReceiverProvider.java#L72
> 
> not removed for now in zipkin

I pointed at the wrong line, I meant the `AcceptorPriorityDelta` config.

Reply:
`AcceptorPriorityDelta ` exists, in line 70, I pointed at the wrong line...

Reply:
> `AcceptorPriorityDelta ` exists, in line 70, I pointed at the wrong line...

Yeah, I mean the config is no longer passed in the actual Armeria server initialization anymore as I can see.

Reply:
@Superskyyy can you remove `AcceptorPriorityDelta` in config file and doc? It's not used anymore in Armeria.

Reply:
Oh, I see thanks @Superskyyy , you can remove it.

Reply:
Sure I will remove it now. haha so confusing the code reference mechanism in GitHub..